target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -225,6 +225,12 @@ glinet,gl-x750)
 hak5,lan-turtle)
 	ucidef_set_led_netdev "wan" "WAN" "orange:system" "eth1"
 	;;
+joyit,jt-or750i)
```

Comment:
I'll check if this is still neeeded.

Reply:
Makes sense (stops the annoying ath10k blinking).

Reply:
The green power/status LED can be controlled by both the SoC and the wireless module, curiously the manufacturer placed series resistors in both paths, so that two brightness levels of green are visible depending on the states of both outputs. We don't know whether this is by error or was even intented by the manufacturer, but the default/demo firmware offered for these boards by Yuncore actually uses the ath10k LED GPIO to signal wireless activity / throughput, though this blinking is quite annoying, considering the LED is very bright and directed upwards, illuminating the case from inside.
http://sebastianschaper.net/openwrt/or750i_leds.png